sdk/python/kubeflow/training/api/test_training_client_create_job.py

```diff
@@ -0,0 +1,194 @@
+import multiprocessing
```

Comment:
Should we name this file as: `training_client_test.py` ? Similar to KFP: https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/client.
We can add more test cases to this file later (e.g. for get_job, list_jobs, etc.)
WDYT @droctothorpe @deepanker13 @johnugeorge ?

Reply:
I prefer the _test.py syntax because it places the test file next to the file being tested when files are sorted alphabetically.

Reply:
+1 on `training_client_test.py`.

Reply:
done